modules/n1ql/pages/n1ql-language-reference/reservedwords.adoc

```diff
@@ -137,34 +144,41 @@ The following keywords are reserved and cannot be used as unescaped identifiers:
 | xref:n1ql-language-reference/union.adoc[INTERSECT]
 | INTO
 | IS
+| xref:n1ql:n1ql-language-reference/set-transaction.adoc[ISOLATION]
 | xref:n1ql-language-reference/createfunction.adoc[JAVASCRIPT]
-| xref:n1ql-language-reference/join.adoc[JOIN]
 
+| xref:n1ql-language-reference/join.adoc[JOIN]
 | KEY
 | KEYS
 | KEYSPACE
 | KNOWN
 | xref:n1ql-language-reference/createfunction.adoc[LANGUAGE]
-| LAST
 
+| LAST
+| LATERAL
 | LEFT
 | xref:n1ql-language-reference/let.adoc[LET]
 | LETTING
 | xref:n1ql:n1ql-language-reference/set-transaction.adoc[LEVEL]
+
 | LIKE
 | xref:n1ql-language-reference/limit.adoc[LIMIT]
-
 | LSM
 | MAP
 | MAPPING
 | MATCHED
+
 | MATERIALIZED
+| MAXVALUE
 | xref:n1ql-language-reference/merge.adoc[MERGE]
-
 | MINUS
```

Comment:
I know it was in the original file, but I don't think `MINUS` shouldn't be here either. Please delete.